### PR TITLE
Improve commit checks and code formatting

### PR DESCRIPTION
**Pull request by rickymoorhouse**

### CODE REVIEW

File: trawler.py [hunk 1]
```diff
@@ -166,28 +168,30 @@ def inc_counter(self, component, target_name, value, pod_name=None, labels=None)
                     metric_name = "{}.{}".format(component, target_name)
                 self.graphite.stage(metric_name, value)
 
+    def is_enabled(self, net_name):
+        """ is net in config and enabled """
+        if net_name in self.config['nets']:
+            if self.config['nets']['certs'].get('enabled', True):
```
Comment (perryan-coder) [inline trawler.py:174]: Is a default of `True` if it cannot find the `enabled` key correct?

File: manager_net.py [hunk 2]
```diff
@@ -98,8 +98,9 @@ def find_hostname(self):
             if self.use_kubeconfig:
                 logger.info("Using KUBECONFIG")
                 config.load_kube_config()
-                netv1 = client.NetworkingV1Api()
-                ingresslist = netv1.list_namespaced_ingress(namespace=self.namespace)
+                v1beta = client.ExtensionsV1beta1Api()
```
[the remark this answers is not in the document]
Comment (rickymoorhouse) [reply manager_net.py:101]: Good catch - I think this was accidental in my rebasing - will remove

File: trawler.py (hunk 1 above)
Comment (rickymoorhouse) [reply trawler.py:174]: Yes that's correct

File: manager_net.py (hunk 2 above)
Comment (rickymoorhouse) [inline manager_net.py:101]:
```suggestion
```

**Review by perryan-coder — APPROVED** [review]

**Pull request merged**